p4-16/spec/P4-16-spec.mdk

```diff
@@ -8619,14 +8619,21 @@ The P4 compiler should provide:
 
 - Errors when annotations are used incorrectly (e.g., an annotation
   expecting a parameter but used without arguments, or with arguments
-  of the wrong type
+  of the wrong type)
 - Warnings for unknown annotations.
 
 # Appendix: Revision History { #sec-revision-history; @h1:"A" }
 
-## Summary of changes made in version 1.2.4
+## Summary of changes made in unreleased version
```

Comment:
What's the intention of this section?

Reply:
The idea will be for PRs that make changes to already add bullet for what they changed. Then, ideally, cutting a new release will be as easy as changing "unreleased version" to "version X.Y.Z"